src/index.html

```diff
@@ -98,6 +102,7 @@ <h3 class='you-vs-others-data activity-data'></h3>
     <script type='text/javascript' src='../data/activity.js'></script>
     <script type='text/javascript' src='./UserRepository.js'></script>
     <script type='text/javascript' src='./HydrationRepository.js'></script>
+    <script type='text/javascript' src='./SleepRepository.js'></script>
     <script type='text/javascript' src='./User.js'></script>
```

Comment:
we are also going to need to add a script element for `ActivityRepository` before we implement anything on the DOM.

Reply:
Will the order be affected? Where should the new line go?